pkg/api/routes_root.go

```diff
@@ -0,0 +1,66 @@
+package api
+
+import (
+	"embed"
+	"html/template"
+	"io/fs"
+	"net/http"
+	"strings"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/stashapp/stash-box/pkg/manager/config"
+)
+
+type rootRoutes struct {
+	ui    embed.FS
+	index []byte
+}
+
+func (rr rootRoutes) Routes() chi.Router {
+	rr.index = getIndex(rr.ui)
+
+	r := chi.NewRouter()
+
+	// session handlers
+	r.Post("/login", handleLogin)
```

Comment:
Note: This is a change from the original code, which used `r.HandleFunc` and returned the index page if `r.Method == http.MethodGet`.
Since login is a POST form, makes more sense to use `r.Post`, and let the other cases fallback to the `/*` handler.